feature_extractor/feature_extractor.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, teams, team_id: int, config: nmmo.config.AllGameSystems):
     self.item_helper = ItemHelper(config, self.entity_helper)
     self.market_helper = MarketHelper(config, self.entity_helper, self.item_helper)
 
+    # TODO: check if using _force_use/sell/buy helps
```

Comment:
could you add a comment with a bit more detail?

Reply:
added more detail like below:
```
    # force_action = True overrides the policy outputs with the featurizer's decisions
    # TODO: check if using featurizer's decisions is actually better
```